docs/src/main/paradox/migration.md

```diff
@@ -74,6 +76,8 @@ The migration tool can be run as main class `akka.persistence.r2dbc.migration.Mi
 
 Durable State is not migrated by `MigrationTool.migrateAll`, instead you need to use `MigrationTool.migrateDurableStates` for a given list of persistence ids.
 
+This list can be retrieved by using `currentPersistenceIds` of @ref:[Durable state queries](./query.md#durable-state-queries). 
```

Comment:
This is misleading because you would have to retrieve them from the source (old) database, and akka-persistence-jdbc doesn't implement that query. Isn't it obvious that you can retrieve such list of persistence ids with just sql select?

Reply:
You are right, I have to remove that sentence as it was a misunderstanding.